### PR TITLE
change alpha-endpoint-reconciler-type argument to endpoint-reconciler-type

### PR DESCRIPTION
**Pull request by rphillips**
**What this PR does / why we need it**: Tweaks the endpoint reconciler argument to remove 'alpha', because according to this [comment](https://github.com/kubernetes/kubernetes/pull/50984#discussion_r146290402) we are preferring to document the flags.

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by ericchiang** [comment]
/ok-to-test

cc @kubernetes/sig-api-machinery-pr-reviews

File: cmd/kube-apiserver/app/options/options.go [hunk 1]
```diff
@@ -166,7 +166,7 @@ func (s *ServerRunOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&s.MasterCount, "apiserver-count", s.MasterCount,
 		"The number of apiservers running in the cluster, must be a positive number.")
 
-	fs.StringVar(&s.EndpointReconcilerType, "alpha-endpoint-reconciler-type", string(s.EndpointReconcilerType),
+	fs.StringVar(&s.EndpointReconcilerType, "endpoint-reconciler-type", string(s.EndpointReconcilerType),
```
Comment (ericchiang) [inline cmd/kube-apiserver/app/options/options.go]: This needs to be deprecated first before being removed. for example https://github.com/kubernetes/kubernetes/pull/51198

Comment (ericchiang) [reply cmd/kube-apiserver/app/options/options.go]: Edit, forgot that this flag wasn't introduced in 1.8 so no need to deprecate anything.

**Comment by rphillips** [comment]
@lavalamp thanks... can you approve no-issue?

**Pull request merged**